File: cpp/program8.cpp
// program98.c

#include<iostream>
using namespace std;

class Pattern
{
    public:
       int iNo;

    Pattern(int X)
    {
        iNo = X;
    }

    void DisplayPattern()
    {
        int iCnt = 0;

        for(iCnt = 0; iCnt <= iNo; iCnt++)
        {
            cout<<iCnt<<"*\t";
        }
    }
};

int main()
{
    int iFrequency = 0;

    cout<<"Enter Frequency "<<"\n";
    cin>>iFrequency;

    Pattern pobj(iFrequency);
    pobj.DisplayPattern();

    return 0;
}

File: scala/program8.scala
import scala.io.StdIn.{readInt}

class Pattern(iNo : Int)
{
    def displayPattern() : Unit  = {
        for(i <- 1 to this.iNo)
        {
            print("*\t")
        }
        println()
    }
}

object Demo 
{    
    def main(args : Array[String])
    {
        println("Enter frequency : ")
        var freq : Int = readInt()

        var obj = new Pattern(freq)
        obj.displayPattern()
    }
}